pkg/controllers/managementuser/nodesyncer/nodessyncer.go

```diff
@@ -826,7 +827,7 @@ func (m *nodesSyncer) isClusterRestoring() (bool, error) {
 	return false, nil
 }
 
-func determineNodeRoles(machine *v3.Node) {
+func determineNodeRoles(machine *v32.Node) {
```

Comment:
Just curious, but what is the *v32 node obj as opposed to the v3?

Reply:
```
	v3 "github.com/rancher/rancher/pkg/generated/norman/management.cattle.io/v3"
	v32 "github.com/rancher/rancher/pkg/apis/management.cattle.io/v3"
```
using `v3.Node` in this case is bad, because `v3.Node` is just defined as the following:

```
// Deprecated: use v3.Node instead
type Node = v3.Node
```
where in the above example, `v3.Node` is actually `v32.Node`, i.e. all api resources defined in `github.com/rancher/rancher/pkg/generated/norman/` are just deprecated aliases to the real API types. The deprecated warnings also show up in my IDE and I hate them, but not in a "I want to turn my warnings off" kind of way, more like a "why are we still using these deprecated types" kind of way. Some people use `mgmtv3`, or `apimgmtv3`, or `v32`. I'm actually just going to update it to be `apimgmtv3`.

Reply:
I see the deprecated warnings too when I use `mgmtv3 Cluster.` The Cluster var is crossed out, not sure why we're still using the generated alias types either